README.md

```diff
@@ -158,6 +158,48 @@ Just want to log violations in production:
 ActionController::Parameters.action_on_invalid_parameters = :log
 ```
 
+## ControllerSupport::ParameterWhitelist
+
+When included into a controller, this module forces the developer
+to explicitly whitelist which parameters are passed to any methods.
+
+Examples:
+
+```ruby
+class TestController < ApplicationController
+  include StrongerParameters::ControllerSupport::ParameterWhitelist
+
+  allow_parameters :show, forum_id: Parameter.integer
+  allow_parameters :create, topic: { forum: { :id: Parameter.integer } }
+  allow_parameters :index, {} # no parameters allowed
+  allow_parameters :update, :anything # all parameters allowed, should only be used when migrating old controllers/actions
+end
+```
+
+The `action_on_invalid_parameters :log` directive may be used to log and warn, but note that it
+is set to raise in tests unless you change `action_on_unpermitted_parameters` (from [Strong Parameters](https://github.com/rails/strong_parameters))
+to `false` or `:log`.
```

Comment:
is this really necessary ... I though we changed that to only log ... don't need to document options we don't need directly